Assets/Scripts/Models/Character/Character.cs

```diff
@@ -577,4 +582,18 @@ private void LoadStats()
 
         Debug.ULogChannel("Character", "Initialized " + stats.Count + " Stats.");
     }
+
+    /// <summary>
+    /// Use the stats of the character to determine various traits.
+    /// </summary>
+    private void UseStats()
+    {
+        // Base character speed on their dexterity.
+        speed = Convert.ToSingle(stats["Dexterity"].Value) / 2;
+
```

Comment:
Actual Leveling doesn't sound quite in line with a basebuilding game, but there might be other things that can affect stats, perhaps training(?) or heavy use to raise a stat, or permanent illness or injury lowering a stat. 

Of course, even barring base stats changing there may be traits that temporarily change a stat (this would probably be a more reasonable way to do illness), in which case speed and other derived stats would still change.

Side Note: this is why I didn't put Health System under 0.2, because it hasn't been heavily discussed and can start getting connected with other systems in a potentially complicated way.